projects/fsastorefrontstyles/scss/components/cms/_find-agent-search-list.scss

```diff
@@ -27,6 +27,11 @@
       }
     }
   }
+  .md-order-change {
```

Comment:
Had to create custom class since BS provides ordering classes control only in one direction, from small to large screen. Here we need it from tablet portrait down.